timeseries/setup.py

```diff
@@ -49,7 +49,7 @@
         "flaky>=3.7,<4",
         "pytest-timeout>=2.1,<3",
         "isort>=5.10",
-        "black>=22.3,<23.0",
+        "black~=23.0",
```

Comment:
I think that `setup.py` for multimodal should be updated as well https://github.com/autogluon/autogluon/blob/master/multimodal/setup.py#L61 .

Maybe there is a way to keep all test dependency versions in one place, similar to https://github.com/autogluon/autogluon/blob/master/core/src/autogluon/core/_setup_utils.py#L18C1-L18C19? Alternatively, maybe we could just put the union of all dependencies necessary for tests in one place and then import/reuse them in each module's `setup.py`?

Reply:
There seems to already be a TODO for that [here](https://github.com/autogluon/autogluon/blob/master/core/src/autogluon/core/_setup_utils.py#L39C1-L39C14).

Reply:
I've updated the missing requirements, I think unifying is a good idea, but should we do it as part of a different PR?